### ISSUE TITLE
when running the forge test it doesn't work Unsupported EVM version

### ISSUE THREAD

**Issue by yongsxyz** [foundry-rs/foundry#4873]
### Component

Forge

### Have you ensured that all of these are up to date?

- [X] Foundry
- [ ] Foundryup

### What version of Foundry are you on?

forge 0.2.0 (e807429 2023-05-03T23:54:27.221626000Z)

### What command(s) is the bug in?

forge test

### Operating System

Windows

### Describe the bug

When running the forge test, I encountered the error message 'Unsupported EVM version' and the test did not work as expected."

```
The application panicked (crashed).
Message:  Unsupported EVM version
Location: cli\src\utils.rs:93
```

**Comment by Evalir** [comment]
Hi! Thanks for reporting this.

Are you setting the `evm_version` in your `foundry.toml`? Could I see the particular test you're trying to run or a minimal reproduction of the issue?

**Comment by yongsxyz** [comment]
I'm sorry, I did not include the 'evm_version' setting in the 'foundry.toml' file. Normally, I do not include this setting and have not experienced any issues like this before. I assure you that this is not a bug. Thank you.

**Comment by Evalir** [comment]
That's alright! Just wanted to confirm if you were setting the `evm_version` manually.

Could we get a minimal repro that causes this behavior based on the tests you're running?

**Comment by yongsxyz** [comment]
> Could we get a minimal repro that causes this behavior based on the tests you're running?

this issue, as usual, I tried to install 'foundry' version forge 0.2.0 (https://github.com/foundry-rs/foundry/commit/e8074297c48eb4343965ca804ef551d10d7b8e72 2023-05-03T23:54:27.221626000Z)  and initialized it by running 'foundry init'. However, when I run the test using the 'forge test' command, I encountered the error message mentioned above.

**Comment by Evalir** [comment]
Interesting—I'm on the same version as you and can't seem to reproduce. What compiler version is it using? I'll also try to test on a Windows box as I'm using macOS.

**Comment by Evalir** [comment]
Hey @yongsxyz — I've tested on a windows box and I'm still not able to reproduce the issue. However, #4856 will soon be merged and should solve this. Unless you object, I'll close the issue once the linked PR is merged.

**Comment by yongsxyz** [comment]
I don't mind, I have found the solution to fix it, as long as I manually set the evm_version in the foundry_toml file. Thank you for the solution provided above.

**Comment by Evalir** [comment]
This is fixed by #4856 — closing
